taipy/gui/gui.py

```diff
@@ -1819,6 +1829,9 @@ def _is_expression(self, expr: str) -> bool:
             return False
         return self.__evaluator._is_expression(expr)
 
+    def _get_variable_dependencies(self, var_name: str) -> t.Set[str]:
```

Comment:
What's the point of this new method (except a shortcut)?

Reply:
coherence with _evaluator access ...

Reply:
Ok...